### ISSUE TITLE
IOS 18.1 beta 5 (22B5054e) support

### ISSUE THREAD

**Issue by papadon6969** [straight-tamago/misakaX#260]
does this support IOS 18.1 beta 5 (22B5054e) support for customization

**Comment by DiegoMA2901** [comment]
Not yet, we're waiting to a new misaka x version to be released and be working with beta 5 of iOS 18

**Comment by xingbadata** [comment]
I don’t believe it would be supported anymore since the exploit has been patched

**Comment by maruyaa** [comment]
When do you think misaka x, which works in IOS18.1 beta5, will release?
